woocommerce/woocommerce-gutenberg-products-block
# Update @wordpress/components to v. 11.1.1 and @wordpress/base-styles to v. 3.2.0

Fixes #2140.
Fixes #3146.
Fixes #3311.
Fixes #3313.

### How to test the changes in this Pull Request:

#### Regression testing

1. Verify you can build the project without problems: `npm ci && npm run build`.
1. Create a page with all WC Blocks (you can copy and paste —with <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>V</kbd>— [this gist](https://gist.github.com/Aljullu/d9c76ed510ae6973bd41dcc4184e8eb2)) + add the Single Product block & some other Product element blocks.
2. In the editor, verify all of them render correctly, you can modify their attributes and there are no errors in the browser console.
3. In the frontend, also verify they render correctly, you can interact with them and there are no errors in the browser console.

#### Testing #3146

1. Go to the Checkout block.
2. Change the value of the Country/Region select.
3. Verify no errors appear in the browser console.
